tensorflow/lite/kernels/internal/reference/space_to_batch_nd.h

```diff
@@ -18,10 +18,9 @@ limitations under the License.
 #include <cmath>
 
 #include "ruy/profiler/instrumentation.h"  // from @ruy
-#include "tensorflow/lite/kernels/internal/types.h"
+#include "tensorflow/lite/kernels/internal/common.h"
```

Comment:
I'm a bit confused by what this commit is fixing. types.h is ok to be included for TFLM.

In fact, the params are declared in that header:
https://github.com/tensorflow/tensorflow/blob/ef214a5a9d217aedcfcff25027d7eab488385e29/tensorflow/lite/kernels/internal/types.h#L1096-L1099

Reply:
underlying issue what elsewhere, and fixed with [`0ec1f46` (#46714)](https://github.com/tensorflow/tensorflow/pull/46714/commits/0ec1f462f82afe9c0e0d5a95958deb71451bc831)